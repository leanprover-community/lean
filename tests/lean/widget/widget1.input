{"seq_num": 0, "command": "sync", "file_name": "widget1.lean"}
{"seq_num": 45, "command": "info", "file_name": "widget1.lean", "line": 66, "column": 2}
{"seq_num": 66, "command": "get_widget", "file_name": "widget1.lean", "line": 66, "column": 0, "id": 456}
{"seq_num": 47, "command" : "widget_event", "handler" : {"h":0, "r" : [457, 456]}, "file_name": "widget1.lean", "line": 66, "column": 0, "id": 456, "args" : { "type" : "string", "value" : "hello world"}}
{"seq_num": 48, "command" : "widget_event", "handler" : {"h":1, "r" : [457, 456]}, "file_name": "widget1.lean", "line": 66, "column": 0, "id": 456, "args" : { "type" : "unit", "value" : {}}}
{"seq_num": 49, "command" : "widget_event", "handler" : {"h":1, "r" : [456]}, "file_name": "widget1.lean", "line": 66, "column": 0, "id": 456, "args" : { "type" : "unit", "value" : {}}}
