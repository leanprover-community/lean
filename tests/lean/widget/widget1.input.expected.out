{"msgs":[{"caption":"trace output","file_name":"widget1.lean","pos_col":0,"pos_line":66,"severity":"information","text":"successfully rendered widget\n"}],"response":"all_messages"}
{"message":"file invalidated","response":"ok","seq_num":0}
{"record":{"widget":{"column":0,"id":456,"line":66}},"response":"ok","seq_num":45}
{"response":"ok","seq_num":66,"widget":{"column":0,"html":{"c":[{"a":{"className":"mw6"},"c":[{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"0"},"c":[{"a":{"className":"dtc v-mid"},"c":["make some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":0,"r":[456]}},"t":"button"}],"t":"div"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"1"},"c":[{"a":{"className":"dtc v-mid"},"c":["delete some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":1,"r":[456]}},"t":"button"}],"t":"div"},{"a":null,"c":[],"t":"hr"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"add row"},"c":[{"c":[{"a":{"className":"dtc v-mid"},"c":[{"a":{"type":"text","value":""},"c":[],"e":{"onChange":{"h":0,"r":[457,456]}},"t":"input"}],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["+"],"e":{"onClick":{"h":1,"r":[457,456]}},"t":"button"}]}],"t":"div"}],"t":"div"}]},"id":456,"line":66}}
{"record":{"status":"success","widget":{"column":0,"html":{"c":[{"a":{"className":"mw6"},"c":[{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"0"},"c":[{"a":{"className":"dtc v-mid"},"c":["make some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":0,"r":[456]}},"t":"button"}],"t":"div"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"1"},"c":[{"a":{"className":"dtc v-mid"},"c":["delete some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":1,"r":[456]}},"t":"button"}],"t":"div"},{"a":null,"c":[],"t":"hr"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"add row"},"c":[{"c":[{"a":{"className":"dtc v-mid"},"c":[{"a":{"type":"text","value":"hello world"},"c":[],"e":{"onChange":{"h":0,"r":[457,456]}},"t":"input"}],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["+"],"e":{"onClick":{"h":1,"r":[457,456]}},"t":"button"}]}],"t":"div"}],"t":"div"}]},"id":456,"line":66}},"response":"ok","seq_num":47}
{"record":{"status":"success","widget":{"column":0,"html":{"c":[{"a":{"className":"mw6"},"c":[{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"0"},"c":[{"a":{"className":"dtc v-mid"},"c":["make some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":0,"r":[456]}},"t":"button"}],"t":"div"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"1"},"c":[{"a":{"className":"dtc v-mid"},"c":["delete some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":1,"r":[456]}},"t":"button"}],"t":"div"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"2"},"c":[{"a":{"className":"dtc v-mid"},"c":["hello world"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":2,"r":[456]}},"t":"button"}],"t":"div"},{"a":null,"c":[],"t":"hr"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"add row"},"c":[{"c":[{"a":{"className":"dtc v-mid"},"c":[{"a":{"type":"text","value":""},"c":[],"e":{"onChange":{"h":0,"r":[457,456]}},"t":"input"}],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["+"],"e":{"onClick":{"h":1,"r":[457,456]}},"t":"button"}]}],"t":"div"}],"t":"div"}]},"id":456,"line":66}},"response":"ok","seq_num":48}
{"record":{"status":"success","widget":{"column":0,"html":{"c":[{"a":{"className":"mw6"},"c":[{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"0"},"c":[{"a":{"className":"dtc v-mid"},"c":["make some tasks"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":0,"r":[456]}},"t":"button"}],"t":"div"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"2"},"c":[{"a":{"className":"dtc v-mid"},"c":["hello world"],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["x"],"e":{"onClick":{"h":1,"r":[456]}},"t":"button"}],"t":"div"},{"a":null,"c":[],"t":"hr"},{"a":{"className":"flex justify-between items-center w-100 bb b--black-20 pb2 mt2","key":"add row"},"c":[{"c":[{"a":{"className":"dtc v-mid"},"c":[{"a":{"type":"text","value":""},"c":[],"e":{"onChange":{"h":0,"r":[457,456]}},"t":"input"}],"t":"div"},{"a":{"className":"dtc v-mid f6 button-reset bg-white ba b--black-20 dim pointer pv1 w2"},"c":["+"],"e":{"onClick":{"h":1,"r":[457,456]}},"t":"button"}]}],"t":"div"}],"t":"div"}]},"id":456,"line":66}},"response":"ok","seq_num":49}
